Makefile

```diff
@@ -225,6 +225,7 @@ release: clean-release ## Builds and push container images using the latest git
 .PHONY: release-manifests
 release-manifests: $(RELEASE_DIR) ## Builds the manifests to publish with a release
 	kustomize build config/default > $(RELEASE_DIR)/cluster-api-components.yaml
+	cat config/certmanager/cert-manager.yaml >> $(RELEASE_DIR)/cluster-api-components.yaml
```

Comment:
Should this be added to `config/default/kustomize.yaml` instead? it seems a bit odd to cat it rather than building it in during the kustomize step. Or is this to avoid overriding the namespace when doing the kustomize build?

Reply:
kustomize adds `capi-` prefix to all cert-manager name/ns :(